### ISSUE TITLE
[question] Device only update status after tapping (off) twice (manual)

### ISSUE THREAD

**Issue by junior466** [zwave-js/zwavejs2mqtt#2249]
I have [this](https://devices.zwave-js.io/?jumpTo=0x0063:0x4944:0x3031:3.37) switch and noticed that it's OFF status will only update in Home Assistant after tapping it twice. So for example, turn on switch manually, HA shows correct status. Turn off switch manually, HA shows switch as ON. Tap off again and HA updates correctly. 

I read about pooling but not sure if this is what I want. If so, what is a good pooling interval to configure? I plan on doing this to a total of 3 devices. I am aware of the consequences but should I worry even if only pooling 3 devices?

**Comment by robertsLando** [comment]
I think it's a device issue as device is not reporting its status, please attach zwavejs logs

**Comment by lee-costa** [comment]
> I think it's a device issue as device is not reporting its status, please attach zwavejs logs

Attached
[zwavejs2mqtt-store.zip](https://github.com/zwave-js/zwavejs2mqtt/files/8025127/zwavejs2mqtt-store.zip)

**Comment by lee-costa** [comment]
And by the way, this also happens when turning on this (and other two switches)

https://user-images.githubusercontent.com/22748735/153018808-fba946e7-cfa4-4774-8505-5c84f7f09cd5.mov

 switch:

**Comment by robertsLando** [comment]
@AlCalzone

**Comment by AlCalzone** [comment]
Which Node ID am I looking for? When (time and timezone) did you interact with it? The log is 77k lines long :)

**Comment by AlCalzone** [comment]
In the time frame you mention I see a couple of these interactions:
```
2022-02-09T23:02:03.600Z DRIVER « [Node 031] [REQ] [ApplicationUpdateRequest]
                                    payload: 0x1f0a04110126277370867277
2022-02-09T23:02:03.601Z CNTRLR « [Node 031] Received updated node info
2022-02-09T23:02:03.601Z CNTRLR   [Node 031] Node does not send unsolicited updates; refreshing actuator and sen
                                  sor values...
2022-02-09T23:02:03.602Z CNTRLR » [Node 031] requesting current switch state...
2022-02-09T23:02:03.604Z DRIVER » [Node 031] [REQ] [SendData]
                                  │ transmit options: 0x25
                                  │ callback id:      166
                                  └─[MultilevelSwitchCCGet]
...
2022-02-09T23:02:03.648Z CNTRLR   [Node 031] [~] [Multilevel Switch] currentValue: 2 => 97          [Endpoint 0]
2022-02-09T23:02:03.654Z DRIVER « [Node 031] [REQ] [ApplicationCommand]
                                  └─[MultilevelSwitchCCReport]
                                      current value: 97
```

Or in plain english:
device: "hey there is something new!"
zwave-js: "okay whats your status?"
device: "I'm currently at level 97"

---

Most of these are just a tiny bit off the limits of fully off (2 or 3 instead of 0) or fully on (97 instead of 99). I think this is a timing issue. The device isn't done dimming when its status is refreshed, so the status ends up as "almost off" instead of off.

We have a compat flag to delay these refreshes, I'm going to try with 250ms which shouldn't feel too long but long enough to avoid the half-dimmed states.

**Comment by AlCalzone** [comment]
@zwave-js-bot add compat flag to 0x0063:0x4944:0x3031:3.37
```jsonc
manualValueRefreshDelayMs: 250
```
